README.md

```diff
@@ -161,14 +161,21 @@ The current status for this install method is *experimental*.
 Please feel free to [report](https://github.com/openPMD/openPMD-api/issues/new/choose) how this works for you.
 
 ```bash
-# optional:             --user
-pip install openpmd-api
+# we need pip 19 or newer
+# optional:                   --user
+python3 -m pip install -U pip
```

Comment:
I usually add the `python3 -m` part of the command to make sure I am installing for the correct python version and I am thinking you did it too. Your change here helps to confirm that we use `Python 3.5 - 3.7` as explained in the build dependencies entry of your [documentation](https://openpmd-api.readthedocs.io/en/0.9.0-alpha/dev/dependencies.html#optional-language-bindings), which is very nice!

Reply:
Yes, exactly. As far as I know it's also the recommend way to use `pip` these days.